run/traffic_split_tag/main.tf

```diff
@@ -25,7 +25,8 @@ resource "google_cloud_run_service" "default" {
     # Update revision to 50% traffic
     percent = 50
     # This revision needs to already exist
-    revision_name = "cloudrun-srv-green"
+    revision = "cloudrun-srv-green"
+    type     = "TRAFFIC_TARGET_ALLOCATION_TYPE_REVISION"
```

Comment:
It might be helpful to link to the enum values. Would https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/cloud_run_v2_service#type be the best link?

Reply:
As above, added.